web/styles/colors.sass

```diff
@@ -0,0 +1,20 @@
+/* colors */
+$white: #FFFFFF
```

Comment:
yaaas we need this - we should also consider having a set variables for spacing as well.

@sherryyx is there currently a spacing schema?

Reply:
(^ not part of this PR) - something to consider going forward